index.php

```diff
@@ -20,47 +20,27 @@
 $query1->execute();
 $result1= $query1->fetchAll();
```

Comment:
You could name this `$result` because I don't think you're using `$result` after you assign `$result [0]['email']` etc to variables...

Here it's not too bad but imagine having `$result` through `$result100`

Also no spacing between variable and square brackets `$result [0]['email'];` => `$result[0]['email'];`

Reply:
I'm using $result higher up on the page for $query and this is part of $query1 so it makes sense.